### PR TITLE
Mock all request to bitly api during specs

### LINKED ISSUE #110: Ensure bitly doesn't get called during spec execution

### PR DESCRIPTION
**Pull request by TheFehr**
Closes #110

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -32,9 +32,13 @@
   config.infer_spec_type_from_file_location!
   config.filter_rails_from_backtrace!
 
-  config.before(:all, type: :system) { Capybara.server = :puma, { Silent: true } }
+  config.before(:all, type: :system) { Capybara.server = :puma, {Silent: true} }
```
Comment (lukasbischof) [inline spec/rails_helper.rb]: Could you not?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -34,3 +35,4 @@
 
   Kernel.srand config.seed
 end
+
```
Comment (lukasbischof) [inline spec/spec_helper.rb]: Y?

**Comment by lukasbischof** [comment]
Plz also run RuboCop

File: spec/rails_helper.rb (hunk 1 above)
Comment (TheFehr) [reply spec/rails_helper.rb]: I did not

**Comment by TheFehr** [comment]
> Plz also run RuboCop

image

File: spec/spec_helper.rb (hunk 2 above)
Comment (TheFehr) [reply spec/spec_helper.rb]: Because Intelij won't let me delete it

**Pull request merged**